public/components/application-ui-tables/8.html

```diff
@@ -0,0 +1,110 @@
+
+<body>
+<div class="overflow-x-auto rounded-lg border border-gray-200">
+  <table class="min-w-full divide-y-2 divide-gray-200 bg-white text-sm" id="user-table">
+    <thead class="ltr:text-left rtl:text-right">
+      <tr>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900 dark:text-white" onclick="sortTable(0)">
+            Name 
+            <span id="name-sort" class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900 dark:text-white" onclick="sortTable(1)">
+            Date of Birth 
+            <span id="name-sort" class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900 dark:text-white" onclick="sortTable(2)">
+            Role 
+            <span id="name-sort" class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="p-2 text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900 dark:text-white" onclick="sortTable(3)">
+            Salary 
+            <span id="name-sort" class="sort-icon">&uarr;&darr;</span>
+        </th>
+      </tr>
+    </thead>
+
+    <tbody class="divide-y divide-gray-200">
+      <tr>
+        <td class="whitespace-nowrap px-4 py-2 font-medium text-gray-900">Stavan Shah</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">14/08/2003</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">Web Developer</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">$120,000</td>
+      </tr>
+
+      <tr>
+        <td class="whitespace-nowrap px-4 py-2 font-medium text-gray-900">Keval Shah</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">17/03/2005</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">Android Developer</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">$100,000</td>
+      </tr>
+
+      <tr>
+        <td class="whitespace-nowrap px-4 py-2 font-medium text-gray-900">Purvesh Shah</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">24/05/1995</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">Singer</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">$150,000</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<script>
```

Comment:
Shouldn't be any JavaScript in base components. There are 3 variants:

- Base (HTML)
- Dark (HTML)
- Interactive (HTML + Alpine JS)

Let's get the base component where it needs to be before tackling anything else.